### PR TITLE
support info command

### LINKED ISSUE #8: Support ngctl info

### PR DESCRIPTION
**Pull request by sixwaaaay**
show information of current Nebula Graph cluster selected by use.

example:
```
+-------------------+-------------------------------+
| Name              | nebula                        |
| Namespace         | default                       |
| CreationTimestamp | 2023-08-13 16:41:18 +0000 UTC |
+-------------------+-------------------------------+
2023/08/13 16:50:06 Overview:
+----------+---------+-------+---------+-----+--------+------------+-----------+---------+
|          | PHASE   | READY | DESIRED | CPU | MEMORY | DATAVOLUME | LOGVOLUME | VERSION |
+----------+---------+-------+---------+-----+--------+------------+-----------+---------+
| Metad    | Running |     1 |       1 | 1   | 1Gi    | 5Gi        | 1Gi       | v3.4.0  |
| Storaged | Running |     3 |       3 | 1   | 1Gi    | 10Gi       | 1Gi       | v3.4.0  |
| Graphd   | Running |     1 |       1 | 1   | 1Gi    |            | 1Gi       | v3.4.0  |
+----------+---------+-------+---------+-----+--------+------------+-----------+---------+
2023/08/13 16:50:06 Endpoints:
2023/08/13 16:50:06 Graphd thrift endpoint: 192.168.49.2:31699
2023/08/13 16:50:06 Graphd http endpoint: 192.168.49.2:30253
2023/08/13 16:50:06 Graphd http2 endpoint: 192.168.49.2:32737
```

fix #8

### CODE REVIEW

**Comment by sixwaaaay** [comment]
need #13

File: cmd/info.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+		return errors.New("no NodePort service found")
+	}
+	for _, port := range service.Spec.Ports {
+		if port.NodePort < 0 {
+			continue
+		}
+		for _, pod := range podList.Items {
+			if pod.Status.Phase == corev1.PodRunning {
+				log.Printf("Graphd %s endpoint: %s:%d", port.Name, pod.Status.HostIP, port.NodePort)
+			}
```
Comment (kqzh) [inline cmd/info.go]: if we use nodePort to connect graphd, the addr is {nodeIp}:{serviceodePort} ?

File: cmd/info.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+		return errors.New("no service found")
+	}
+	service := svc.Items[0]
+	if service.Spec.Type != corev1.ServiceTypeNodePort {
+		return errors.New("no NodePort service found")
+	}
+	for _, port := range service.Spec.Ports {
+		if port.NodePort < 0 {
+			continue
+		}
```
Comment (kqzh) [inline cmd/info.go:160]: if service't type is clusterIP, how to get endpoint?

File: cmd/info.go [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	t.AppendRow(table.Row{"Metad",
+		status.Metad.Phase, workload.ReadyReplicas, *metad.Replicas,
+		metad.Resources.Limits.Cpu(), metad.Resources.Limits.Memory(), metad.DataVolumeClaim.Resources.Requests.Storage(),
+		metad.LogVolumeClaim.Resources.Requests.Storage(), metad.Version})
+
+	workload = status.Storaged.Workload
+	storaged := spec.Storaged
+	t.AppendRow(table.Row{"Storaged",
+		status.Storaged.Phase, workload.ReadyReplicas, *storaged.Replicas,
+		storaged.Resources.Limits.Cpu(), storaged.Resources.Limits.Memory(), storaged.DataVolumeClaims[0].Resources.Requests.Storage(),
```
Comment (kqzh) [inline cmd/info.go]: if  storaged have multi DataVolumeClaims, need print them all ?

**Review by kqzh — APPROVED** [review]
LGTM

**Pull request merged**